packages/compass-indexes/src/components/indexes-table/indexes-table.tsx

```diff
@@ -84,13 +73,15 @@ export const IndexesTable: React.FunctionComponent<IndexesTableProps> = ({
   onDeleteIndex,
 }) => {
   const columns = useMemo(() => {
-    const _columns = [
-      'Name and Definition',
-      'Type',
-      'Size',
-      'Usage',
-      'Properties',
-    ].map((name) => {
+    const _columns = (
+      [
+        'Name and Definition',
+        'Type',
+        'Size',
+        'Usage',
+        'Properties',
+      ] as SortColumn[]
+    ).map((name) => {
```

Comment:
tiny TS nit, feel free to ignore: `as` does a cast and would accept array entries here that aren’t actually valid values for `SortColumn`. You could avoid that issue through e.g.:

```suggestion
    const sortColumns: SortColumn[] = [
      'Name and Definition',
      'Type',
      'Size',
      'Usage',
      'Properties',
    ];
    const _columns = sortColumns.map((name) => {
```